agenta-web/src/components/Evaluations/Evaluations.tsx

```diff
@@ -27,7 +27,6 @@ import EvaluationErrorModal from "./EvaluationErrorModal"
 import {getAllVariantParameters} from "@/lib/helpers/variantHelper"
 
 import Image from "next/image"
-import abTesting from "@/media/testing.png"
```

Comment:
Please put this back

Reply:
I have effected the changes.

But why did it fail if it's unused in this file?
Or was it later used by other devs? Shouldn't that have raised merge conflicts?